<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HCDelegateMainWindow</class>
 <widget class="QMainWindow" name="HCDelegateMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HCDelegate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioSerialPort">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTCP">
        <property name="text">
         <string>TCP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="linePort">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
